packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -137,13 +137,22 @@ describe('ReactDOMFloat', () => {
     buffer = '';
   }
 
-  function getVisibleChildren(element) {
+  function getVisibleChildrenAndResources(element) {
     const children = [];
     let node = element.firstChild;
     while (node) {
       if (node.nodeType === 1) {
         if (
-          node.tagName !== 'SCRIPT' &&
+          // Only include scripts with src elements, this will hide instruction scripts
+          !(node.tagName === 'SCRIPT' && !node.hasAttribute('src')) &&
```

Comment:
We plan on running all tests for both the external runtime and inline runtime. So this distinction might not make sense.

Also, we should probably have a few intentional inline scripts tests too.

Reply:
I suppose I could look at async too but distinguishing between component provided inline scripts and instruction scripts (non-external) seems tricky